vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java

```diff
@@ -18,12 +22,13 @@
 public class MongoClientOptionsParser {
 
   private final MongoClientSettings settings;
+  private final static CodecRegistry commonCodecRegistry = CodecRegistries.fromCodecs(new StringCodec(), new IntegerCodec());
```

Comment:
I had a brief attempt at distinct and I ended up with the list:
```
                    new IntegerCodec(),
                    new BooleanCodec(),
                    new DoubleCodec(),
                    new BooleanCodec(),
                    new LongCodec(),
                    new StringCodec(),
                    new ByteCodec(),
                    new FloatCodec(),
                    new BooleanCodec()
```

Is there a reason to only add int and string?